src/placeos-frontend-loader/api/repositories.cr

```diff
@@ -28,16 +26,17 @@ module PlaceOS::FrontendLoader::Api
     end
 
     def self.commits(folder : String, branch : String, count : Int32 = 50, loader : Loader = Loader.instance)
-      with_query_directory(folder, loader) do |key, directory|
-        Git.repository_commits(
-          repository: key,
-          working_directory: directory,
-          count: count,
-          branch: branch
-        )
+      HashFile.config({"base_dir" => "#{loader.content_directory}/#{folder}/metadata"})
```

Comment:
Instead of changing the config every time you want to access it, it would be better to have a single file, and the values namespaced.

Reply:
So going `store.metadata(folder)` would be behind the scenes going `HashFile["${folder}/metadata"]`
Doing this would encourage consolidation of metadata in a single file, and keep its accesses encapsulated in the store object